changelog/add_i18nlocaletext_match_for_redirect_back.md

```diff
@@ -0,0 +1 @@
+* [#880](https://github.com/rubocop/rubocop-rails/pull/880): Add Rails/I18nLocaleTexts match for redirect_back. ([@bensheldon][])
```

Comment:
The following test fails.

```console
% bundle exec rake spec
(snip)

rspec './spec/project_spec.rb[1:2:4:1:7]' # RuboCop Rails Project Changelog future entries For /Users/koic/src/github.com/rubocop/rubocop-rails/spec/../changelog/add_i18nlocaletext_match_for_redirect_back.md starts with `new_`, `fix_`, or `change_`
```

Can you rename this filename to `changelog/change_add_i18nlocaletext_match_for_redirect_back.md`?

Reply:
Oops! Just renamed and force-pushed.